src/main/java/org/apache/commons/fileupload/servlet/ServletRequestContext.java

```diff
@@ -123,5 +123,14 @@ public String toString() {
                 Long.valueOf(this.contentLength()),
                 this.getContentType());
     }
+    
+     /**
+     *
+     * @return
```

Comment:
This method is not used in the PR.

Reply:
this method is to be used in subclasses of FileUploadBase to get request for accessing request attributes.